profiles/permissions.py

```diff
@@ -1,10 +1,13 @@
 """
 Permission classes for profiles
 """
+from django.http import Http404
+from rest_framework.generics import get_object_or_404
 from rest_framework.permissions import (
     BasePermission,
     SAFE_METHODS,
 )
+from profiles.models import Profile
```

Comment:
Can you add an empty line before here? Imports should be grouped into three groups with a empty line separating each group: https://www.python.org/dev/peps/pep-0008/#imports